easybuild/easyconfigs/c/Clang/Clang-8.0.0-GCCcore-8.2.0.eb

```diff
@@ -51,6 +58,11 @@ builddependencies = [
 
 assertions = True
 usepolly = True
+build_lld = True
+libcxx = True
+enable_rtti = True
+default_openmp_runtime = 'libomp'
```

Comment:
As mentioned in https://github.com/easybuilders/easybuild-easyblocks/pull/1717#issuecomment-492983499 , turns out this is actually the default since 2015 so might as well remove this.